versions/2.0/vipps-login-migrate-api-1.0-to-2.0.md

```diff
@@ -0,0 +1,28 @@
+# Vipps Login API
+
+Document version 1.0.0.
+
+Reference:\
+[vipps-login-api-version-1.0](../../vipps-login-api)\
+[vipps-login-api-version-2.0](vipps-login-api.md)
+
+## Table of contents
+* [Overview](#overview)
+* [Convert to vipps login api version 2.0](#convert-to-vipps-login-api-version-20)
+
+## Overview:
```

Comment:
Could we add a _short_ description on why we are doing the changes.

Reply:
Updated the text with a short description. Please take another look as I have rewritten the text for better flow.